Mathlib/Algebra/Algebra/Equiv.lean

```diff
@@ -23,6 +23,7 @@ This file defines bundled isomorphisms of `R`-algebras.
 universe u v w u₁ v₁
 
 /-- An equivalence of algebras is an equivalence of rings commuting with the actions of scalars. -/
+@[stacks 09HR]
```

Comment:
I don't think this tag is right.

Reply:
09HR writes:
Let E/F be an extension of fields. Then Aut(E/F) or AutF(E) denotes the automorphism group of E as an object of the category of F-extensions. Elements of Aut(E/F) are called automorphisms of E over F or automorphisms of E/F.

In mathlib, we do use `AlgEquiv` for Aut(E/F)?

Reply:
I added a comment 
"This generalize beyond the field case in 09HR. In mathlib, we use `AlgEquiv` to denote Aut(E/F)."

Reply:
`AlgEquiv` is more than `Aut(E/F)`.  
I think this tag should be moved to the group instance on `AlgEquiv F E E`.

Reply:
Yes. I'll do it.

Reply:
Fixed.